### PR TITLE
chore: [SONIC-169] main ← 2u/project-replatforming

### PR DESCRIPTION
**Pull request by pshiu**
## Description

Rebase main ← 2u/project-replatforming.

## Additional Information

* Jira: [SONIC-169](https://2u-internal.atlassian.net/browse/SONIC-169) Merge: Coordinator's 2u/project-replatforming into master to be deployed on stage and on prod

## Testing Information

`manage.py runserver` and `manage.py celery` start normally on my local.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,40 +2,42 @@ name: Python CI
 
 on:
   push:
-    branches: [main]
+    branches: 
+      - main
+      - 2u/project-theseus
+      - 2u/project-replatforming
```
Comment (shafqatfarhan) [inline .github/workflows/ci.yml:8]: Can we remove this feature branch now from the CI run list?

File: commerce_coordinator/apps/commercetools/tests/raw_ct_order.json [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+  "returnInfo": [
+  ],
+  "lastMessageSequenceNumber": 7,
+  "lastModifiedBy": {
+    "clientId": "E1UbNz9SHIG_n4nnSAUP1uW4"
+  },
+  "createdBy": {
+    "clientId": "E1UbNz9SHIG_n4nnSAUP1uW4"
+  }
+}
```
Comment (shafqatfarhan) [inline commerce_coordinator/apps/commercetools/tests/raw_ct_order.json:321]: EOF line missing - might error our in linting

**Review by shafqatfarhan — APPROVED** [review]
💯

File: .github/workflows/ci.yml (hunk 1 above)
Comment (pshiu) [reply .github/workflows/ci.yml]: Fixed in 80a41b58431e2a6af8e9cd1bb700a61cb3a8cee0!

File: commerce_coordinator/apps/commercetools/tests/raw_ct_order.json (hunk 2 above)
Comment (pshiu) [reply commerce_coordinator/apps/commercetools/tests/raw_ct_order.json]: Fixed in f7c34d46ddfec481c86b2ca24872cb1b460d7f72!

**Pull request merged**